b.tsv

```diff
@@ -587,10 +587,10 @@ B50	Sicilian Defense: Modern Variations, Tartakower	1. e4 c5 2. Nf3 d6 3. d4 cxd
 B50	Sicilian Defense: Wing Gambit, Deferred Variation	1. e4 c5 2. Nf3 d6 3. b4
 B51	Sicilian Defense: Canal Attack, Dorfman Gambit	1. e4 c5 2. Nf3 d6 3. Bb5+ Nc6 4. O-O Bd7 5. Qe2 g6 6. e5
 B51	Sicilian Defense: Canal Attack, Moscow Gambit	1. e4 c5 2. Nf3 d6 3. Bb5+ Nc6 4. O-O Bd7 5. c3 Nf6 6. Re1 a6 7. Bxc6 Bxc6 8. d4 Bxe4 9. Bg5
-B51	Sicilian Defense: Moscow Variation	1. e4 c5 2. Nf3 d6 3. Bb5+
+B51	Sicilian Defense: Canal Attack,	1. e4 c5 2. Nf3 d6 3. Bb5+
```

Comment:
can you move up this line to match the naming order. also the `,` is not needed

Reply:
Thank you